cli/src/commands/git/clone.rs

```diff
@@ -252,23 +266,29 @@ fn fetch_new_remote(
     ui: &Ui,
     workspace_command: &mut WorkspaceCommandHelper,
     remote_name: &RemoteName,
+    target_branch: Option<RefNameBuf>,
```

Comment:
nit: `Option<&RefName>`

Reply:
Changed to `Option<&[StringPattern]>` now that multiple branches are supported.